Sources/SpeziDevicesUI/Measurements/MeasurementRecordedSheet.swift

```diff
@@ -0,0 +1,221 @@
+//
+// This source file is part of the Stanford SpeziDevices open source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ACarousel
+import HealthKit
+import OSLog
+@_spi(TestingSupport) import SpeziDevices
+import SpeziViews
+import SwiftUI
+
+
+/// A sheet view displaying a newly recorded measurement.
+///
+/// This view retrieves the pending measurements from the ``HealthMeasurements`` Module that is present in the SwiftUI environment.
+public struct MeasurementRecordedSheet: View {
```

Comment:
Maybe now that we support multiple measurements, it should be named `MeasurementsRecordedSheet`?
```suggestion
public struct MeasurementsRecordedSheet: View {
```